tests/testparse.cpp

```diff
@@ -404,4 +404,89 @@ void TestParse::testParseMerge()
 		SUBSURFACE_TEST_DATA "/dives/mergedVyperOstc.xml");
 }
 
+int TestParse::parseCSVmanual(int units, std::string file)
+{
+	verbose = 1;
+	char *params[55];
+	int pnr = 0;
+
+	params[pnr++] = strdup(strdup("numberField"));
```

Comment:
This "feature" was implemented in 2015-08-23, so gotta fix also in parseCSV() function.